packages/vm/package.json

```diff
@@ -19,7 +19,7 @@
     "test:state:slow": "npm run test:state -- --runSkipped=slow",
     "test:buildIntegrity": "npm run test:state -- --test='stackOverflow'",
     "test:blockchain": "node -r ts-node/register --stack-size=1500 ./tests/tester --blockchain",
-    "test:API": "tape -r ts-node/register --stack-size=1500 ./tests/api/**/*.js",
+    "test:API": "tape -r ts-node/register --stack-size=1500 './tests/api/**/*.js'",
```

Comment:
that's funny adding quotes fixes, it interpolates it in a different way? in another branch I used a solution you suggested to me by specifying both `./tests/api/*.js ./tests/api/**/*.js`

Reply:
Glob expansion is generally confusing, as each system or package that parses it can have a different behavior. That's exactly the same thing with the `~/` in a path, that caused problems in Grid in the past :)

These work:
- `./tests/api/*.js ./tests/api/**/*.js`. but I also needed to include `./tests/api/**/*/*.js` to account for three-level depth.
- `'./tests/api/**/*.js'`

This didn't:
- `./tests/api/**/*.js`